src/main/views/courts/tabs/types.njk

```diff
@@ -0,0 +1,11 @@
+{% from "macros/csrf.njk" import csrfProtection %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+
+<div id='courtTypesTab'>
+  <input id='slug' type='hidden' value='{{court.slug}}'></input>
```

Comment:
I think you can delete this hidden input - it should already be in the edit-court-general.njk file

Reply:
removed hidden input